travis.sh

```diff
@@ -1,13 +1,4 @@
 #!/bin/bash
 
-set -euo pipefail
```

Comment:
@inverno keep this one. It should fail the build fast

Reply:
In fact, I just removed `travis.sh`